packages/kotti-ui/source/kotti-field-file-upload/components/TakePhoto/utils/common.ts

```diff
@@ -0,0 +1,47 @@
+import dayjs from 'dayjs'
+
+const ISO8601 = 'YYYY-MM-DD HH:mm:ss' as const
```

Comment:
this should be moved to kotti-ui's global constants if possible

I think we had a similar constant for the date fields (maybe unify both of them)

Reply:
Moved to kotti-ui's global constants

But cannot use in date fields bc they don't use ISO8601 format